Index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE >
+<html lang="en">
+
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <title>Protfolio Mobile-setup</title>
+
+  <!-- Load Styles' file -->
+  <link href="styles.css" rel="stylesheet">
+
+  <!-- Load Fonts -->
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header_section">
+
+    <img src="Sources/Union.png" alt="Unicon vector" id="unicon">
```

Comment:
Please consider wrapping the hamburger icon of the mobile menu using the semantic `nav` tag instead of the generic `div`. Using the semantic `nav` makes the user experience of users using web accessibility tools like screen readers more pleasant.